### PR TITLE
add DID Resolution to charter scope

### LINKED ISSUE #24: Add DID Resolution to charter scope
During our F2F meeting in Vancouver, many participants voiced support for beginning to specify DID Resolution.

### PR DESCRIPTION
**Pull request by brentzundel**
closes #24 

Signed-off-by: Brent Zundel <brent.zundel@gmail.com>

***
<a href="https://pr-preview.s3.amazonaws.com/brentzundel/did-wg-charter/pull/25.html" title="Last updated on Oct 20, 2022, 7:40 PM UTC (bff750a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-wg-charter/25/13cc943...brentzundel:bff750a.html" title="Last updated on Oct 20, 2022, 7:40 PM UTC (bff750a)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -80,7 +80,7 @@ <h1 id="title"><i class="todo">PROPOSED</i> Decentralized Identifier Working Gro
 <a href="https://www.w3.org/2019/did-wg/">Decentralized Identifier Working Group</a>
 is to maintain the <a href="https://www.w3.org/TR/did-core/">Decentralized Identifiers (DIDs)</a>
 specification and related Working Group Notes. The WG will also seek consensus
-around the specification of DID Methods.
+around the specification of DID Methods and DID Resolution.
```
Comment (peacekeeper) [inline index.html:83]:
```suggestion
around the specification of DID Methods and DID Resolution (incl. DID URL Dereferencing).
```

File: index.html [hunk 2]
```diff
@@ -152,6 +152,9 @@ <h1 id="title"><i class="todo">PROPOSED</i> Decentralized Identifier Working Gro
 The Working Group will also seek consensus around DID Method specifications, and
 may produce related documents.
         </p>
+        <p>
+The Working Group will begin working toward a specification for DID Resolution.
```
Comment (peacekeeper) [inline index.html:156]:
```suggestion
The Working Group will begin working toward a specification for DID Resolution (incl. DID URL Dereferencing).
```

File: index.html [hunk 3]
```diff
@@ -178,8 +181,8 @@ <h2>Scope</h2>
         <p>The Working Group may also publish new Working Group Notes that may be necessary to clarify, help the deployment and usage, or propose new features to the Group’s main recommendation-track deliverable.</p>
         <p>
 The Working Group may also publish new Working Group Notes, Editors Drafts, or
-Working Drafts of DID Method Specifications. Any documents intended for
-Recommendation will be limited to the Working Draft maturity level.
+Working Drafts for DID Resolution or of DID Method Specifications. Any documents
```
Comment (peacekeeper) [inline index.html:184]:
```suggestion
Working Drafts for DID Resolution (incl. DID URL Dereferencing) or of DID Method Specifications. Any documents
```

**Review by peacekeeper — APPROVED** [review]
Yes, this is good. I would just propose that the specification, even though it may be named "DID Resolution", should also include "DID URL Dereferencing" in its scope.

File: index.html (hunk 1 above)
Comment (TallTed) [reply index.html:83]: Best not to use abbreviations like `incl.`  Please change all three to `including`, or consider defining `DID Resolution` to include `DID URL Dereferencing` up front (which would make the subsequent parenthetical `including` notes redundant and unnecessary).

**Review by peacekeeper — APPROVED** [review]

File: index.html [hunk 4]
```diff
@@ -181,8 +182,9 @@ <h2>Scope</h2>
         <p>The Working Group may also publish new Working Group Notes that may be necessary to clarify, help the deployment and usage, or propose new features to the Group’s main recommendation-track deliverable.</p>
         <p>
 The Working Group may also publish new Working Group Notes, Editors Drafts, or
-Working Drafts for DID Resolution or of DID Method Specifications. Any documents
-intended for Recommendation will be limited to the Working Draft maturity level.
+Working Drafts for DID Resolution, DID URL Dereferencing, or of DID Method
```
Comment (TallTed) [inline index.html:185]: This change could be expanded to as `Working Drafts for DID Resolution or for DID URL Dereferencing, or of DID Method`
```suggestion
Working Drafts for DID Resolution or DID URL Dereferencing, or of DID Method
```

File: index.html (hunk 1 above)
Comment (brentzundel) [reply index.html:83]: addressed in [409f1c6](https://github.com/w3c/did-wg-charter/pull/25/commits/409f1c61b8000805fe33cb67e8e8ee43ad8bf25a)

File: index.html (hunk 2 above)
Comment (brentzundel) [reply index.html]: addressed in [409f1c6](https://github.com/w3c/did-wg-charter/pull/25/commits/409f1c61b8000805fe33cb67e8e8ee43ad8bf25a)

File: index.html (hunk 3 above)
Comment (brentzundel) [reply index.html]: addressed in [409f1c6](https://github.com/w3c/did-wg-charter/pull/25/commits/409f1c61b8000805fe33cb67e8e8ee43ad8bf25a)

**Review by OR13 — APPROVED** [review]

**Review by burnburn — APPROVED** [review]

**Review by mprorock — APPROVED** [review]
This approach seems sane to me

**Pull request merged**